dts/bindings/kscan/nuvoton,npcx-kscan.yaml

```diff
@@ -0,0 +1,32 @@
+# Copyright (c) 2022 Nuvoton Technology Corporation.
+# SPDX-License-Identifier: Apache-2.0
+
+description: Nuvoton, NPCX keyboard matrix controller
+
+compatible: "nuvoton,npcx-kscan"
+
+include: kscan.yaml
+
+properties:
+    reg:
+        required: true
+
+    clocks:
+        required: true
+
+    pinctrl-0:
+        type: phandles
+        required: true
+        description: configurations of pinmux controllers
```

Comment:
gentle reminder: a pinctrl driver has to be implemented, this will be deprecated soon. https://github.com/zephyrproject-rtos/zephyr/issues/39740

Reply:
Thanks for the reminder, Nuvoton has already provided an PR  proposal #43638 for the Pinctrl driver.
We will refactor all npcx drivers to use the Pinctrl API once the driver is accepted.